daemon/commands.c

```diff
@@ -510,7 +511,13 @@ static void command_thread(void *arg)
         command_thread_error = ret;
         goto error_after_pipe_bind;
     }
-    if ((ret = uv_listen((uv_stream_t *)&server_pipe, SOMAXCONN, connection_cb))) {
+    ret = uv_listen((uv_stream_t *)&server_pipe, SOMAXCONN, connection_cb);
+    if (ret) {
+        /* Fallback to backlog of 1 */
+        info("uv_listen() failed with backlog = %d, falling back to backlog = 1.", SOMAXCONN);
+        ret = uv_listen((uv_stream_t *)&server_pipe, 1, connection_cb);
+    }
+    if (ret) {
```

Comment:
This second check is necessary only case we `fallback to backlog of 1`, should we not move it for inside the previous `if`?

Reply:
The logic is identical, I did not want to nest too much to improve readability. But if this confuses you, maybe readability is bad after all, hmm.

Reply:
No, it does not create any confusion while I am reading, I only requested, because case `ret` was tested to zero before, it won't change late. It was a requested related to performance.

Reply:
In that case I prefer it to stay as it is, because I think readability is more important in this case, cause this is only called 1 time during initialization, which is at most 1 per day if you update daily. So I don't think performance is important here.

Reply:
I understood, this is the motive that I marked as `resolved`. I am testing the PR now to be sure that everything is right, I will approve it in few hours case there is not any problem.